core/src/player.rs

```diff
@@ -177,9 +177,17 @@ struct GcRootData<'gc> {
     /// Dynamic root for allowing handles to GC objects to exist outside of the GC.
     dynamic_root: DynamicRootSet<'gc>,
 
+    #[allow(clippy::type_complexity)]
```

Comment:
Is this allow really needed up here? `Vec<PostFrameCallback<'gc>>` is a very simple type